support/ebpf/hotspot_tracer.ebpf.c

```diff
@@ -787,30 +817,30 @@ ErrorCode hotspot_read_codeblob(const UnwindState *state, const HotspotProcInfo
   // the exact CodeBlob/CompiledMethod/nmethod size. The CodeBlob is allocated in the JIT area,
   // preceding the actual JIT code and data for the function. It is thus exceedingly unlikely for
   // us to accidentally read into a guard / unallocated page despite the over-read.
-  if (bpf_probe_read_user(scratch->codeblob, sizeof(scratch->codeblob), (void*)cbi->address)) {
+  if (bpf_probe_read_user(scratch->codeblob, sizeof(scratch->codeblob), (void *)cbi->address)) {
     goto read_error_exit;
   }
 
   // Make the verifier happy. No bound checks required for the remaining offsets: they are u8, and
   // the verifier is aware that their maximum value is smaller than our `codeblob` buffer.
-  if (ji->nmethod_deopt_offset   + sizeof(u64) > sizeof(scratch->codeblob) ||
-      ji->nmethod_compileid      + sizeof(u32) > sizeof(scratch->codeblob) ||
+  if (ji->nmethod_deopt_offset + sizeof(u64) > sizeof(scratch->codeblob) ||
+      ji->nmethod_compileid + sizeof(u32) > sizeof(scratch->codeblob) ||
       ji->nmethod_orig_pc_offset + sizeof(u64) > sizeof(scratch->codeblob)) {
     return ERR_UNREACHABLE;
   }
 
   // Extract the needed CodeBlob fields.
-  cbi->code_start     = *(u64*)(scratch->codeblob + ji->codeblob_codestart);
-  cbi->code_end       = *(u64*)(scratch->codeblob + ji->codeblob_codeend);
-  cbi->frame_size     = *(u32*)(scratch->codeblob + ji->codeblob_framesize) * 8;
-  cbi->frame_comp     = *(u32*)(scratch->codeblob + ji->codeblob_framecomplete);
-  cbi->compile_id     = *(u32*)(scratch->codeblob + ji->nmethod_compileid);
-  cbi->orig_pc_offset = *(u32*)(scratch->codeblob + ji->nmethod_orig_pc_offset);
-  cbi->deopt_handler  = *(u64*)(scratch->codeblob + ji->nmethod_deopt_offset);
+  cbi->code_start = *(u64 *)(scratch->codeblob + ji->codeblob_codestart);
+  cbi->code_end = *(u64 *)(scratch->codeblob + ji->codeblob_codeend);
+  cbi->frame_size = *(u32 *)(scratch->codeblob + ji->codeblob_framesize) * 8;
+  cbi->frame_comp = *(u32 *)(scratch->codeblob + ji->codeblob_framecomplete);
+  cbi->compile_id = *(u32 *)(scratch->codeblob + ji->nmethod_compileid);
+  cbi->orig_pc_offset = *(u32 *)(scratch->codeblob + ji->nmethod_orig_pc_offset);
+  cbi->deopt_handler = *(u64 *)(scratch->codeblob + ji->nmethod_deopt_offset);
```

Comment:
The previous alignment of `= ...` pleases my eyes. Not sure if clang-format supports that, though.

Reply:
Added and applied AlignConsecutiveAssignments: AcrossComments with https://github.com/open-telemetry/opentelemetry-ebpf-profiler/pull/310/commits/ca7c86e0be6d557188c4cda2e34c88575c9dc880.